src/core/storage/fileio/union_fstream.cpp

```diff
@@ -76,7 +73,11 @@ union_fstream::union_fstream(std::string url,
     } catch(...) {
       log_and_throw_io_failure("Unable to open " + url);
     }
+#else
```

Comment:
corresponding to comment above; I think we should explicitly tell users (well, us) the download failure is due to no remote fs compilation, instead of bug in our code base.

Reply:
I think we do.  If it doesn't we should resolve that.

Reply:
good to know!